integration/config/msp/admincerts/admincert.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
same thing here, any reason not to create these keys/certificates at build time?

Reply:
this is inherited from run auction.  I guess we could create them but i'd hope that whether we have the same or fresh fabric credential shouldn't really affect anything we do -- we are not using them directly -- and static ones are easier to maintain ...